woocommerce/woocommerce-android
# The new "learn more" button on the connection dialog

Closes: #7182 


### Description

The PR adds "learn more" button to the scanning state of the connection to a reader dialog

### Testing instructions

* Start IPP collection flow
* Notice that there is a new "learn more" section
* Click on it and verify that web view with information about IPP is open

### Images/gif


https://user-images.githubusercontent.com/4923871/185381580-e631e596-3846-4d06-9b8c-c72a1c75d97d.mp4
